Fix Compatiblity with Python 3.5

RobertBlackhart opened the pull request:
Commit piccolomo/plotext@63125c9 added an f-string in _global.py which breaks the stated Python3.5 compatibility from setup.py. This change should make it compatible again by using .format() on the string instead of an f-string.

RobertBlackhart commented:
Alternatively, Python 3.5 support can be dropped since that release is no longer supported (only 3.7 and up are currently in a supported state).

piccolomo commented:
Thanks! Will be involved in this soon. I guess the `pt` lambda function is not used anyway if I remember.

RobertBlackhart commented:
I added a second commit that fixes a problem when trying to run under PyPy. The `_global` module was trying to import itself and that lead to an AttributeError, but only when run in PyPy. It worked fine in cPython. However, I didn't see a need for it to happen so I removed it which should let it work in either.

piccolomo commented:
Thanks!
